main.py

```diff
@@ -31,7 +31,7 @@ def main(cfg: DictConfig):
     utils.check_base_results_saved(wandb.config.model.path, wandb.config.unadapt)
 
     logger.info("Loading model")
-    model_base = models.load_model(wandb.config.model, logger)
+    model_base = models.load_model(wandb.config.model, wandb.config.device)
```

Comment:
Could 'device' be stored under model in the config, like `wandb.config.model.device`? Seems cleaner that way. Also, good to pass in the logger so we can log issues or special notes about loading the model.